src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -129,7 +129,7 @@ const ReportActionItemFragment = (props) => {
                         >
                             <Text
                                 selectable={false}
-                                style={[styles.w1, styles.userSelectNone]}
+                                style={[EmojiUtils.containsOnlyEmojis(text) ? styles.onlyEmojisTextLineHeight : undefined, styles.w1, styles.userSelectNone]}
```

Comment:
@kushu7 I can see `EmojiUtils.containsOnlyEmojis(text)` already being called in the parent `Text` `EMOJI_TEXT_FONT`. Can we take this method outside the return to be called only once, and the boolean result at both the places?

Reply:
Sure